MobiFlight/MobiFlightCache.cs

```diff
@@ -87,7 +87,7 @@ public void Start()
                     if (0 == AvailableComModules.Count) {
                         isFirstTimeLookup = false;
                         LookupFinished?.Invoke(this, new EventArgs());
-                        Log.Instance.log($"End looking up connected modules. No modules found.", LogSeverity.Debug);
+                        Log.Instance.log($"Finish looking up connected modules. No modules found.", LogSeverity.Info);

```

Comment:
I changed these events to info because I think they might be valuable enough to be displayed together with the other board detection events.